human_male_female/MD5SUMS

```diff
@@ -0,0 +1,17 @@
+eb321ecd19ebe81ea0c8b88a969cd5fd  GRCh38/Homo_sapiens.GRCh38.dna.chromosome.22.fa.gz
+974059b6a79eeed96ccc8ee9d36d9e8e  GRCh38/Homo_sapiens.GRCh38.dna.chromosome.X.fa.gz
+c4d00789488f974a30f9bdf80718940b  GRCh38/Homo_sapiens.GRCh38.dna.chromosome.Y.fa.gz
+2349c4332beb2dec7dd4bdf74dbe2309  reference.fa
```

Comment:
Shouldn't the `.fai` and sequence dict also be included?

Reply:
:+1: Will create those as well.